raitools/raitools/_managers/explainer_manager.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+"""Defines the Explainer Manager class."""
+
+import warnings
+from scipy.sparse import issparse
+from interpret_community.mimic.mimic_explainer import MimicExplainer
+from interpret_community.mimic.models.lightgbm_model import (
+    LGBMExplainableModel)
+from interpret_community.mimic.models.linear_model import (
+    LinearExplainableModel)
+from interpret_community.common.constants import ModelTask
+from raitools._managers.base_manager import BaseManager
+
+SparseNumFeaturesThreshold = 1000
+
+
+class ExplainerManager(BaseManager):
```

Comment:
If managers are going to be public we should deliberate a bit on the name of explainability/interpret that we want to use for all of these things. One reason to prefer interpret is that counterfactuals are also considered explanations. The distinction might start to get confusing if we don't have good names for these two components.

Reply:
agree, but FeatureExplainerManager seems too long, and interpret is never used in code by industry, it's only used in product names.  Counterfactual Explanations are different from Feature Importance Explanations, but since the latter is more prevalent I would argue to keep it as just Explainer?